src/parsers/trades/liquid/index.ts

```diff
@@ -0,0 +1,88 @@
+import { getCSVData } from '../../';
+import { EXCHANGES, IImport, IPartialTrade, ITrade } from '../../../types';
+import { createDateAsUTC, createID } from '../../utils';
+
+interface ILiquid {
+    exchange: string;
+    currency_type: string;
+    direction: string;
+    transaction_id: string;
+    transaction_type: string;
+    gross_amount: string;
+    currency: string;
+    execution_id: string;
+    generated_for_type: string;
+    generated_for_id: string;
+    transaction_hash: string;
+    from_address: string;
+    to_address: string;
+    state: string;
+    created_at_jst: string;
+    updated_at_jst: string;
+    created_at_utc: string;
+    updated_at: string;
+    notes: string;
+    used_id: string;
+    account_id: string;
+}
+
+export default async function processData(importDetails: IImport): Promise<ITrade[]> {
+    const data: ILiquid[] = await getCSVData(importDetails.data) as ILiquid[];
+    const internalFormat: ITrade[] = [];
+    if (data.length < 1) {
```

Comment:
Thoughts on moving line 30-34 into its own small function for all the parsers?

I feel like we just keep repeating this for all of them. Doesnt have to be in this PR but just curious what your thoughts are on it

Reply:
See https://github.com/starsoccer/cryptotithe/pull/281/files#r1047200157

Reply:
^ that link is not working for me for some reason

Reply:
That’s because I changed the whole liquid code in between